modules/pattern/pattern.service.js

```diff
@@ -29,14 +32,14 @@ class PatternsService {
   }
 
   async checkPatternExist(data) {
-    const pattern = await Pattern.find({
+    const pattern = await Pattern.countDocuments({
```

Comment:
Please, call variable properly, because it is not an actual entity, it is a count